README.md

```diff
@@ -17,13 +17,13 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
-[] Undefined
+[X] Strings
+[X] Booleans
+[X] Undefined
 [] NaN
 [] Integers
```

Comment:
Something minor to add, Integers are a type of `Number` which is a primitive data type in JS. Also, Arrays are an example of a complex data type in JS